README.md

```diff
@@ -22,6 +22,10 @@ Then add a new repository with the URL:
 
 Alternatively, the plugin can be installed by cloning the repository and placing it in your QGIS plugin folder (e.g. C:\Users\username\.qgis2\python\plugins on Windows)
 
+## Configuration
+
+The branding and target web service URL are parsed from a config file (plugin.cfg) in the directory of the plugin.
+
 ## Support
 
 [OSMA Web Services Wiki](http://wms.locationcentre.co.uk/wiki/index.php/OSMA_Web_Services)
```

Comment:
http://osmahelp.themapcloud.com is now set up so it should be safe to use this as the URL for now.